### PR TITLE
Bugfix get maintenance status

### PR DESCRIPTION
**Pull request by pawbu**
I got Internal server error on first request to GET /maintenance-status. This PR fixes the issue by returning the same object also when `initialize` method runs.

### CODE REVIEW

**Review by vaskalan — APPROVED** [review]
LGTM!

File: tests/unit/adapter/test_maintenance_db.py [hunk 1]
```diff
@@ -15,65 +15,81 @@ def teardown_module():
 
 
 def setup_function():
-    DB_CLIENT.jobdb.drop_collection("maintenance")
+    DB_CLIENT.maintenance_db.drop_collection("maintenance")
```
Comment (DanielElisenberg) [inline tests/unit/adapter/test_maintenance_db.py:18]: In the adapter implementation we use db.collection = jobDB.maintenance collection 🤔 That should ideally be reflected here too in order to not be confusing

File: job_service/adapter/maintenance_db.py [hunk 2]
```diff
@@ -29,14 +29,15 @@ def set_status(status_request: MaintenanceStatusRequest):
             "paused": status_request.paused,
             "timestamp": str(datetime.now()),
         }
-        doc_id = maintenance.insert_one(document)
-        return document
+        doc_id = maintenance.insert_one(document).inserted_id
+        return maintenance.find_one({"_id": doc_id}, projection={"_id": False})
```
Comment (DanielElisenberg) [inline job_service/adapter/maintenance_db.py:33]: :100:

**Review by DanielElisenberg — COMMENTED** [review]
:100: This looks like a more reasonable implementation👍🏻 One question about the db.collection definitions in the tests 👀

File: tests/unit/adapter/test_maintenance_db.py (hunk 1 above)
Comment (pawbu) [reply tests/unit/adapter/test_maintenance_db.py:18]: That is true, I'll fix

**Review by DanielElisenberg — APPROVED** [review]
Looks good :100:
This does not affect the test cases at all, but as a side note it seems that mongo takes case into account for collection names. If we have a jobDB already created and I try to insert something into a collection under jobdb:

`pymongo.errors.WriteError: db already exists with different case` 

Again, this does not affect the mocked test collections at all, but I found it worth mentioning 👍🏻

**Review by vaskalan — APPROVED** [review]
Looks good!

**Pull request merged**